adapters/adprime/adprime.go

```diff
@@ -0,0 +1,142 @@
+package adprime
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/buger/jsonparser"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// AdprimeAdapter struct
+type AdprimeAdapter struct {
+	URI string
+}
+
+// NewAdprimeBidder Initializes the Bidder
+func NewAdprimeBidder(endpoint string) *AdprimeAdapter {
+	return &AdprimeAdapter{
+		URI: endpoint,
+	}
+}
+
+type adprimeParams struct {
```

Comment:
Sorry we missed this in our previous review but looks like since you're using `jsonparser` to parse the `tagID` from the bidder ext, you don't need this struct. I don't see it being referenced anywhere either.